sapl/protocoloadm/email_utils.py

```diff
@@ -0,0 +1,205 @@
+from datetime import datetime as dt
```

Comment:
Talvez possamos mover esse arquivo `email_utils.py` para o `sapl/base` ou o código para o arquivo `utils.py`. Daí teria que adaptar a parte de envio do email daqui e de matéria legislativa para usar essa biblioteca em comum, i.e., menos copy-and-paste. O que vc acha?

Reply:
Ótima ideia! Farei isso